examples/pytorch/token-classification/run_ner.py

```diff
@@ -202,6 +209,30 @@ def __post_init__(self):
                 assert extension in ["csv", "json"], "`validation_file` should be a csv or a json file."
         self.task_name = self.task_name.lower()
 
+class MyCallback(TrainerCallback):
+    "A callback log loss, learning rate, and throughput each logging step"
+    start_time = time.time()
+
+    def on_step_begin(self, args: TrainingArguments, state: TrainerState, control: TrainerControl, **kwargs):
+        # count the time after the logging step
+        if state.global_step == 0 or state.global_step % args.logging_steps == 1:
+            self.start_time = time.time()
+        
+    def on_log(self, args: TrainingArguments, state: TrainerState, control: TrainerControl,**kwargs):
+        if args.logging_strategy == 'steps':
+            logging_step_runtime = time.time() - self.start_time
+            num_samples = args.per_device_train_batch_size * args.logging_steps
+            throughput = num_samples / logging_step_runtime
+            state.log_history[-1]["throughput"] = throughput
```

Comment:
While you're at it, why don't you work on printing out loss, lr, and evaluation metrics as well?

Reply:
But, loss and lr also should be printed out at every log interval. That's written in [Benchmark Code Guideline](https://www.notion.so/Benchmark-Code-Guideline-1c544796906a4ccea337ae289dab5b68?pvs=4).

Reply:
And evaluation metrics are expected to be logged at the evaluation phase.